models/src/main/java/com/vimeo/networking2/NotificationConnection.kt

```diff
@@ -0,0 +1,51 @@
+package com.vimeo.networking2
+
+/**
+ * Requires [CapabilitiesType.CAPABILITY_APP_NOTIFICATIONS].
+ */
+data class NotificationConnection(
+
+    /**
+     * The total number of new notifications. This data requires a bearer token
+     * with the private scope.
+     */
+    val newTotal: Int? = null,
+
+    /**
+     * The total number of notifications. This data requires a bearer token with
+     * the private scope.
+     */
+    val total: Int? = null,
+
+    /**
+     * The total number of new notifications. This data requires a bearer
+     * token with the private scope.
+     */
+    val totalNew: Int? = null,
```

Comment:
There is a `newTotal` and `totalNew` property both with the same docs. Are they duplicates of each other?